core/client/app/controllers/application.js

```diff
@@ -25,23 +25,23 @@ export default Ember.Controller.extend({
     }),
 
     actions: {
-        topNotificationChange (count) {
+        topNotificationChange: function (count) {
             this.set('topNotificationCount', count);
         },
 
-        toggleAutoNav () {
+        toggleAutoNav: function () {
```

Comment:
Related: http://nodejsreactions.tumblr.com/post/99325652420/discussing-the-merits-of-new-es6-features